TbsCore/Core/TaskTimer.cs

```diff
@@ -41,41 +43,51 @@ public void Stop()
 
         private async void NewTick()
         {
-            try
-            {
-                if (acc.Tasks.Count == 0) return; //No tasks
+            if (acc.Tasks.Count == 0) return; //No tasks

```

Comment:
there were some exceptions before, are they gone so we can remove try/catch?

Reply:
what are they? i didn't see anything except exception related with browser

Reply:
no idea, I just remember it was throwing errors that's why I added try/catch